README.md

```diff
@@ -1 +1,37 @@
-# cosign-installer
\ No newline at end of file
+# cosign-installer GitHub Action
+
+This action enables you to sign, verify container images. For a quick start of `cosign`, please refer to https://github.com/sigstore/cosign#quick-start
+
+## Usage
+
+Add the following entry to your Github workflow YAML file with the required inputs:
```

Comment:
This sentence says that the `cosign-release` input is required, but it isn't _strictly_ required -- and further down the README, we claim that it's optional as well. Clarifying the wording here would be helpful.

Reply:
done, thanks for your review and feedback